docker/debian-base.dockerfile

```diff
@@ -8,21 +8,21 @@ WORKDIR /app
 # Install Curl
 # Install Apprise, add sqlite3 cli for debugging in the future, iputils-ping for ping, util-linux for setpriv
 # Stupid python3 and python3-pip actually install a lot of useless things into Debian, specify --no-install-recommends to skip them, make the base even smaller than alpine!
-RUN apt update && \
-    apt --yes --no-install-recommends install python3 python3-pip python3-cryptography python3-six python3-yaml python3-click python3-markdown python3-requests python3-requests-oauthlib \
-        sqlite3 iputils-ping util-linux dumb-init git && \
+RUN apt-get update && \
```

Comment:
Any reason why changed to `apt-get`?

Reply:
Using plain `apt` outputs a warning about the lack of a stable interface.  It's not meant to be used in scripts basically, but apt-get is stable and outputs no warning.